src/components/Home.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import { fetchCrypto } from '../redux/crypto/crypto';
+import Loading from './Loading';
+
+const Home = () => {
+  const [search, setSearch] = useState('');
+  const coins = useSelector((state) => state.cryptoData);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCrypto());
+  }, [dispatch]);
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const filteredCoins = coins.crypto.filter((coin) => (
+    coin.name.toLowerCase().includes(search.toLowerCase())
+  ));
+  if (coins.isLoding || coins.isEror) {
+    return <Loading />;
+  }
+
+  return (
+    <header className="home-container">
+      <div className="home-serch">
+        <input
+          type="text"
+          value={search}
+          placeholder="Search Coins..."
+          onChange={handleSearch}
+        />
+      </div>
+      <div className="home-list">
+        {filteredCoins.length === 0 ? (
+          <h1>No related coins found!</h1>
+        ) : (
+          filteredCoins.map((coin) => (
+            <div key={coin.id} className="container-card">
+              <img src={coin.icon} alt={coin.name} />
+              <div className="container-card-info">
+                <h1>{coin.name}</h1>
+                <p>
+                  Price: $
+                  {coin.price.toFixed(2)}
+                </p>
+              </div>
+              <NavLink to={`/details/${coin.id}`}>
+                <FaArrowAltCircleRight className="link" />
+              </NavLink>
+            </div>
+          ))
+        )}
+      </div>
```

Comment:
- [OPTIONAL] Nice and beautiful design so far, however, as stated in the project requirements, you are suppose to follow the design guidelines including: 

- [x] Colors (select one main color for your website).
- [x] Typography: font face, size and weight.
- [x] Layout: composition and space between elements.

There are some differences between yours and that of the actual design which you will need to fix.

1.  The home page is actually suppose to be in two rows with a light and dark color alternating between the cards on each column.  
2. Also, there are no spaces between the cards
3. The app should also have one color covering the background of the whole app as shown below.


### Your design 

![Screenshot from 2023-02-08 00-16-45](https://user-images.githubusercontent.com/65233860/217398115-3b93e01c-dad1-4308-8a9f-0c22be5df9a6.png)


### The actual design 

![Screenshot from 2023-02-07 18-49-55](https://user-images.githubusercontent.com/65233860/217398126-e4f1d1b4-53cf-4f6c-b918-6e8cf6854c94.png)